samples/webcodecs-echo/js/main.js

```diff
@@ -258,7 +317,37 @@ document.addEventListener('DOMContentLoaded', async function(event) {
     // WritableStream of VideoFrames, using non-standard Chrome API
     const generator = new MediaStreamTrackGenerator({kind: 'video'});
     outputStream = generator.writable;
-    document.getElementById('outputVideo').srcObject = new MediaStream([generator]);
+    outputVideo.srcObject = new MediaStream([generator]);
+
+    // Initialize variables
+    let paint_count = 0;
+    let start_time = 0.0;
+
+    const doSomething = (now, metadata) => {
```

Comment:
should this function be called "recordStartTime"?

Reply:
This function is obtaining metadata from remote Video (specifically the presentationTime of the output video stream) so I renamed it to recordOutputFrames.